Cargo.toml

```diff
@@ -20,7 +20,7 @@ smallvec = { version = "1", default-features = false }
 socket2 = { version = "0.4", optional = true, default-features = false }
 tokio = { version = "1", default-features = false }
 # Disable default-features to exclude unused dependency on libudev
-tokio-serial = { version = "5.4.0-beta4", optional = true, default-features = false }
+tokio-serial = { version = "5.4.1", optional = true, default-features = false }
```

Comment:
I would suggest to pin only the major.minor version 5.4 without specifiying the patch version. This is how we handle all other dependencies.

@flosse?

Reply:
Fine with me. Feel free to change that as needed, you should have commit rights on my PR branch.

Reply:
Done: 60e069a2c1d7114a0bd0ecd9b584bcb9258bc8d2